src/Components/SignupLogin/Login.js

```diff
@@ -16,7 +19,12 @@ const Login = () => {
 
           <Form.Group controlId="passWord">
             <Form.Label>Password</Form.Label>
-            <Form.Control type="password" placeholder="Password" />
+            <InputGroup>
+              <Form.Control type={showPassword ? 'text' : 'password'}  placeholder="Password" />
+              <InputGroup.Append>
+                <Button variant="secondary" onClick={handleShowPassword}>{showPassword ? 'Hide' : 'Show'}  </Button>
```

Comment:
Use react-icon (BsEye) instead of Button.

Reply:
@vaishali614 done.